app/models/report.rb

```diff
@@ -166,10 +166,33 @@ def badge_values(variant)
       color = "red"
     end
 
-    ["#{word} closed in", duration_in_words.downcase, color]
+    ["#{word} closure", duration_in_words.downcase, color]
   end
 
   def metadata_attrs
     %i(open_issues_count stargazers_count forks_count size language description)
   end
+
+  # Get the approximate disntance of time in words from the given from_time
+  # to the the given to_time. If to_time is not specified then it is set
+  # to 0.
+  def approximate_distance_of_time_in_words(from_time, to_time=0)
```

Comment:
Assignment Branch Condition size for approximate_distance_of_time_in_words is too high. [21.66/15]<br>Surrounding space missing in default value assignment.